client/src/App.vue

```diff
@@ -240,4 +243,10 @@ export default class App extends Mixins(SessionTimerMixin) {
   opacity: 0.46
   background-color: rgb(33, 33, 33) // grey darken-4
   border-color: rgb(33, 33, 33) // grey darken-4
+
+.app-footer
+  .container
+    max-width: 100%
```

Comment:
Is this because of the div that the header is within (`<div id="main-column">`) ?

If so, would it make sense to move the footer outside this div?

Reply:
I tried this option. its not fixing the issue